pkg/extensions/search/cve/cve_test.go

```diff
@@ -727,8 +727,6 @@ func TestCVEConfig(t *testing.T) {
 
 		secondDir := t.TempDir()
 
-		CopyTestFiles("../../../../test/data", path.Join(secondDir, "a"))
-
 		ctlr.Config.Storage.RootDirectory = firstDir
```

Comment:
It seems to me like the purpose of this test was to enable the search, and obtain coverage on:
https://github.com/project-zot/zot/commit/b0ed625a2e0f7c5724a6d3c613c066ac874dbc87

However the CVE extensions/configuration was removed and the test was repurposed to test multiple storage subpaths in:
https://github.com/project-zot/zot/commit/28974e81dcd449e78d4735f08b960b88f5ea068a#diff-5e37dc387d048f301d803af010fae652fc9fb27118226fd3145ed4372fb22900L707

I suggest we:
- rename `TestCVEConfig` to `TestCVEMultipleSubstores`
- we should either add images with CVEs (keep CopyTestFiles and scan these images), or mock the scanner.
- add some calls at the end of the tests to verify the CVEs present in the images using the graphql API.

Reply:
Commented this test and I will check how this affects coverage and if it can be removed or has to be modified